public/src/js/models/collections/collection.js

```diff
@@ -268,10 +268,11 @@ export default class Collection extends BaseClass {
             reportErrors(new Error(message)); //report to sentry
         })
         .catch(() => {
+            const breakingNewsMsg = 'breaking news alert. The alert may not have been sent. Please contact Central Production for more information.';
```

Comment:
`The alert may not have been sent.` - it would be nice to be more definite here. Do we have any way of knowing here whether it has been sent, or is there always going to be some uncertainty?

Reply:
It definitely would -- but I don't think we can know without understanding all the different failure modes of our backend and the notification service. Our backend has validation, and the notification service will have its own checks to run before it sends the notification. In addition to this, the notification service will have different error states, too. 

I suspected this work would take a long time, so rather than give a categorical answer, I've left it ambiguous in the hopes that CP will let users know more, with the understanding that we could refine this approach if needed. Perhaps that isn't the right approach, though -- I'd be interested to know your thoughts.

Reply:
I think this is an improvement on the existing experience already, so happy to approve on that basis.

Reply:
When does this `catch` get exercised? Is it if we've failed to write to App's SNS topic?

Reply:
It happens when our call to the collection action returns an error code. A failure to send a breaking news alert is one way this can happen -- but there are many others, too, I think, as this this code handles publish and discard actions and there are validation steps before each action, too.

Reply:
Hmm, on closer inspection I think we can be pretty confident that an error would mean an alert wasn't sent -- I thought that a notifications client was handling the alert sending, but we're just posting to an endpoint. With that in mind I've changed the wording to remove the ambiguity @jennygrahamjones @akash1810